dotCMS/src/integration-test/java/com/dotcms/security/apps/AppsUtilTest.java

```diff
@@ -0,0 +1,159 @@
+package com.dotcms.security.apps;
+
+import static com.dotcms.security.apps.AppsUtil.bytesToCharArrayUTF;
+import static com.dotcms.security.apps.AppsUtil.charsToBytesUTF;
+import static com.dotcms.security.apps.AppsUtil.readJson;
+import static com.dotcms.security.apps.AppsUtil.toJsonAsChars;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.util.Logger;
+import com.liferay.util.Encryptor;
+import com.liferay.util.EncryptorException;
+import com.tngtech.java.junit.dataprovider.DataProvider;
+import com.tngtech.java.junit.dataprovider.DataProviderRunner;
+import com.tngtech.java.junit.dataprovider.UseDataProvider;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.security.Key;
+import java.util.Map.Entry;
+import java.util.Set;
+import org.apache.commons.lang.RandomStringUtils;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(DataProviderRunner.class)
+public class AppsUtilTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        IntegrationTestInitService.getInstance().init();
+        SecretsStore.INSTANCE.get().deleteAll();
+    }
+
+    /**
+     * Data provider to feed byte conversion test.
+     * @return
+     * @throws Exception
+     */
+    @DataProvider
+    public static Object[] getTestCases() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5329810)